src/lib/commands.ts

```diff
@@ -16,11 +16,11 @@ export function getCategories(): Record<
 > {
   return {
     apis: {
-      description: 'Upload OpenAPI/Swagger definitions',
+      description: 'OpenAPI / Swagger',
```

Comment:
wayyyy cleaner of a header, good lordt

Reply:
the amount of little oversights on the **main** help screen that we don't notice for literal years lmao

Reply:
🤡